samza-yarn/src/main/java/org/apache/samza/job/yarn/YarnContainerRunner.java

```diff
@@ -190,13 +197,25 @@ private void startContainer(Path packagePath,
       throw new SamzaContainerLaunchException("IO Exception when writing credentials to output buffer");
     }
 
+    Map<String, LocalResource> localResourceMap = new HashMap<>();
+    localResourceMap.put("__package", packageResource);
+
+    // include the resources from the universal resource configurations
+    try {
+      LocalizerResourceMapper resourceMapper = new LocalizerResourceMapper(config, yarnConfiguration);
```

Comment:
Would n't it be nicer to standardize all localizer related configs in one-place? Currently, they appear to be in here, as well as in `LocalizerResourceMapper`. I wonder if we can consolidate all of them into a single function that takes in a config , and returns a map of local resources we can directly pass in to `SamzaContainerContext`.?

Something along the lines of :
```
public Map<String, LocalResource> getLocalResources(Config config) {

}
```
You can use the existing `LocalizerResourceMapper` class, or provide a `static` function (in one of the util classes)